### ISSUE TITLE
cors_domain is not being read

### ISSUE THREAD

**Issue by dwolff** [thomaxxl/safrs#3]
Even if you set the "cors_domain" variable, it is not being read on the jsonapi.py, via globals(). I'm using python 2.7.14

**Comment by thomaxxl** [comment]
Ack, I'll get this fixed asap.

**Comment by thomaxxl** [comment]
I made a demo app for cors: https://github.com/thomaxxl/safrs/blob/master/examples/demo_cors.py . Unfortunately it only seems to work for python3.

**Comment by dwolff** [comment]
Indeed, I was having some problems POSTing new items to the database, and it was solved upgrading the environment to Python 3.x.

That means that the library is not Python 2.x compatible, as stated on the README.

**Comment by dwolff** [comment]
I've also found out that, if you use the "DELETE" verb, when using CORS only, it throws the following message:

TypeError: Object of type 'Response' is not JSON serializable
